integrated_channels/blackboard/admin/__init__.py

```diff
@@ -0,0 +1,40 @@
+"""
+Admin integration for configuring Blackboard app to communicate with Blackboard systems.
+"""
+
+from django.contrib import admin
+
+from integrated_channels.blackboard.models import BlackboardEnterpriseCustomerConfiguration
+
+
+@admin.register(BlackboardEnterpriseCustomerConfiguration)
+class BlackboardEnterpriseCustomerConfigurationAdmin(admin.ModelAdmin):
+    """
+    Django admin model for BlackEnterpriseCustomerConfiguration.
+    """
+    list_display = (
+        "enterprise_customer_name",
+        "client_id",
+        "client_secret",
+        "blackboard_base_url",
```

Comment:
I haven't touched the admin pages here much, but why is enterprise_customer_name listed in `list_display` and readonly_fields, but `refresh_token` is only in read only?

Reply:
hmm, only 98% on this, but I believe enterprise_customer_name also needs to be shown in the changelist page which shows all the configs whereas the refresh token only displays once you select a config?